src/main/java/fr/insee/lunatic/model/flat/Roundabout.java

```diff
@@ -0,0 +1,57 @@
+package fr.insee.lunatic.model.flat;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Roundabout component. Its behavior is similar to a loop, but with refinements.
+ * The core idea is to display a menu (the "roundabout") for the iterations.
+ */
+@Getter
+@Setter
+public class Roundabout extends ComponentType implements ComponentNestingType {
+
+    /**
+     * Object that holds the configuration of roundabout's items.
+     */
+    @Getter
+    @Setter
+    public static class Item {
+
+        /** Label of the items collected in the roundabout. */
+        @JsonProperty(required = true)
+        private LabelType label;
+
+        /** Optional description of the items collected in the roundabout. */
+        private LabelType description;
+
+        /** Expression that determines if a roundabout's item has to receive answers or not. */
+        private LabelType disabled;
+    }
+
+    public Roundabout() {
+        super();
+        this.componentType = ComponentTypeEnum.ROUNDABOUT;
+        this.components = new ArrayList<>();
+    }
+
+    /** Expression that defines the number of items in the roundabout. */
+    private LabelType iterations;
+
+    /** Boolean option to lock or not an item answers when it gets the 'completed' status. */
+    private Boolean locked;
```

Comment:
Cette propriété n'est pas utilisé dans Lunatic me semble.

Reply:
Pourtant elle apparaît dans le [type source](https://github.com/InseeFr/Lunatic/blob/825b7096760bf3d866fad7f52508b3e233cca30b/src/type.source.ts#L54) et [l'exemple du storybook](https://github.com/InseeFr/Lunatic/blob/825b7096760bf3d866fad7f52508b3e233cca30b/src/stories/roundabout/source.json#L70), si j'ai bien compris le cas d'usage est de ne pas permettre au répondant de revenir sur une itération dont le statut serait "complétée" (en utilisant la progress variable sans doute).

Côté conception / génération, le travail est fait pour générer cette propriété, je propose de la laisser pour l'instant, ensuite 

- soit on l'exploite dans Lunatic
- soit le métier n'en veut plus finalement et tant pis (tant mieux ? ^^) on jette

Qu'en penses-tu ?